docs/management/managing-tags.asciidoc

```diff
@@ -0,0 +1,70 @@
+[role="xpack"]
+[[managing-tags]]
+== Tags
+
+Tags enable you to categorize your saved objects. You can then easily filter for related objects based on shared tags.
+
+To begin, open the main menu, click *Stack Management*, then click *Tags*.
+
+[role="screenshot"]
+image::images/tags/tag-management-section.png[Tags management section]
+
+[float]
+=== Required permissions
+
+Access to *Tags* requires the `Tag Management` {kib} privilege. To add the privilege, open the menu, 
+click *Stack Management*, then click *Roles*.
+
+In addition:
+
+* The `read` privilege allows you to assign tags to the saved objects for which you have write permission.
+* The `write` privilege enables you to create, edit, and delete tags.
+
+
+NOTE: having the `Tag Management` {kib} privilege is not required to
+view tags assigned on objects the user has `read` access to, or to filter objects by tags
+in {kib} applications or from the navigational search.
+
+[float]
+[[settings-create-tag]]
+=== Create a tag
+
+Create a tag to assign to your saved objects.
+
+. Click *Create tag*.
+
+. Enter a name and select a color for the new tag.
+
+. Click *Create tag*.
+
+[role="screenshot"]
+image::images/tags/create-tag.png[Tag creation popin]
+
+[float]
+[[settings-assign-tag]]
+=== Assign a tag to saved objects
+
+Assign or remove tags to one or more saved objects.
+
+. Click the action (...) icon in the tag row, and then select the *Manage assignments* action.
+. Select the objects to which you want to assign or remove tags.
+. Click on *Save tag assignments*.
+
+[role="screenshot"]
+image::images/tags/manage-assignments-flyout.png[Assign flyout]
+
+TIP: To assign multiple tags to objects at once, select their checkboxes
```

Comment:
although I had to add another image, what do you think about showing a screenshot with the checkboxes checked?